ui/app/src/components/PreviewBrowseComponentsPanel/PreviewBrowseComponentsPanel.tsx

```diff
@@ -296,60 +201,46 @@ interface BrowsePanelUIProps {
-              classes={{ root: classes.pagination, selectRoot: 'hidden', toolbar: classes.toolbar }}
-              component="div"
-              labelRowsPerPage=""
-              count={count}
-              rowsPerPage={limit}
-              page={pageNumber}
-              backIconButtonProps={{
-                'aria-label': formatMessage(translations.previousPage),
-                size: 'small'
-              }}
-              nextIconButtonProps={{
-                'aria-label': formatMessage(translations.nextPage),
-                size: 'small'
-              }}
-              onPageChange={(e: React.MouseEvent<HTMLButtonElement>, page: number) => onPageChanged(e, page * limit)}
-            />
-          </div>
-          <List className={classes.list}>
-            {items.map((item: ContentInstance) => (
-              <DraggablePanelListItem
-                key={item.craftercms.id}
-                primaryText={item.craftercms.label}
-                onDragStart={() => onDragStart(item)}
-                onDragEnd={onDragEnd}
-              />
-            ))}
-          </List>
-          {count === 0 && (
-            <EmptyState
-              title={formatMessage(translations.noResults)}
-              classes={{ image: classes.noResultsImage, title: classes.noResultsTitle }}
-            />
-          )}
-        </>
-      ) : (
-        <EmptyState
-          title={formatMessage(translations.chooseContentType)}
-          image={chooseContentTypeImageUrl}
-          classes={{
-            root: classes.emptyState,
-            image: classes.emptyStateImage,
-            title: classes.emptyStateTitle
+      <div className={classes.paginationContainer}>
+        <TablePagination
+          className={classes.pagination}
+          classes={{ root: classes.pagination, selectRoot: 'hidden', toolbar: classes.toolbar }}
+          component="div"
+          labelRowsPerPage=""
+          count={count}
+          rowsPerPage={limit}
+          page={pageNumber}
+          backIconButtonProps={{
+            'aria-label': formatMessage(translations.previousPage),
+            size: 'small'
           }}
+          nextIconButtonProps={{
+            'aria-label': formatMessage(translations.nextPage),
+            size: 'small'
+          }}
+          onPageChange={(e: React.MouseEvent<HTMLButtonElement>, page: number) => onPageChanged(e, page * limit)}
+        />
+      </div>
+      <List className={classes.list}>
+        {items.map((item: ContentInstance) => (
+          <DraggablePanelListItem
+            key={item.craftercms.id}
+            primaryText={item.craftercms.label}
+            onDragStart={() => onDragStart(item)}
+            onDragEnd={onDragEnd}
+          />
+        ))}
+      </List>
+      {count === 0 && (
+        <EmptyState
+          title={formatMessage(translations.noResults)}
+          classes={{ image: classes.noResultsImage, title: classes.noResultsTitle }}
         />
```

Comment:
When there aren't any results, is there an empty message elsewhere? 
I'd also like to add a note, that only shared components are shown here. Let's discuss.

Reply:
there is an <EmptyState/>